app/controllers/v1/forams_controller.rb

```diff
@@ -16,8 +16,10 @@ def index
 
       respond_to do |format|
         format.json { paginate json: forams }
-        format.csv  { render csv: forams }
-        format.gen  { render gen: forams }
+        format.all do
```

Comment:
Dobra, `xml` to głupi przykład bo jest metoda `to_xml`. Ale jak ktoś zrobi `.../forams.html` to już poleci 500